engine/examples/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/googollee/go-socket.io/engine"
+)
+
+func main() {
+	eio := engine.New(engine.DefaultOption().
```

Comment:
By @ralfbawg example this code transformed into:
```go
eio := engine.New(
                WithPingInterval(25 * time.Second),
		WithPingTimeout(50 * time.Second),
	)
``` 
Create with default options:
```go
eio := engineio.New()
```